charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -1243,6 +1249,37 @@ agent:
         }
       }
     }
+  windowsDefaultConfig:
```

Comment:
Dont think this property is used anywhere.. we should remove it

Reply:
yep this should be removed